thirdeye/thirdeye-frontend/app/styles/components/rootcause-chart.scss

```diff
@@ -1,4 +1,8 @@
 .rootcause-chart {
   flex-grow: 1;
   min-height: 400px;
+
+  &--split__header {
```

Comment:
Since this is an element and not a modifier, to comply to BEM standards, it should be `__split-header`, since `--` is reserved for modifiers like color or position. 
https://en.bem.info/methodology/naming-convention/#two-dashes-style